ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java

```diff
@@ -1,19 +1,29 @@
 /**
  * Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * A copy of the License is located at
- *
- *     http://aws.amazon.com/apache2.0/
- *
+ * <p>
+ * http://aws.amazon.com/apache2.0/
+ * <p>
  * or in the "license" file accompanying this file. This file is distributed
  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
 package com.amazon.android.contentbrowser;
 
+import android.app.Activity;
```

Comment:
Could we avoid all those code formatting changes and displacements of original code blocks? They will complicate update to new Fire App Builder release and merging the original code with our changes in future .

Reply:
please check the latest PR, formatting issues are resolved. thanks